modules/libprotobuf-mutator/1.3/patches/add_build_file.patch

```diff
@@ -0,0 +1,23 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,20 @@
++licenses(["notice"])  # Apache 2
++
++cc_library(
++    name = "libprotobuf-mutator",
```

Comment:
This will be called `liblibprotobuf-mutator.so`, could you add the usual `alias`?